### ISSUE TITLE
TargetEveryOne guys claim our benchmark against MariaDB be unfair

### ISSUE THREAD

**Issue by miyconst** [Starcounter/Starcounter#3107]
@moander from TargetEveryOne says that our benchmarking against MariaDB is not fair, because NodeJs is a super slow platform for such things, and MariaDB should be tested with c# instead.

cc @warpech, @dan31, @Starcounter-Jack

**Comment by k-rus** [comment]
cc @linkdata

**Comment by linkdata** [comment]
Node.js is a popular (and quite fast) framework, and it's MySQL driver is one of it's most tested components. We could certainly have tested more frameworks, but in the time given we had to choose just one. The test is well documented and on github, so anyone can repeat it with other frameworks.

I am a bit surprised hearing that "NodeJs is a super slow platform for such things". Especially since we saw it scale almost perfectly with reads, and the performance degradation from writes to a multi-machine database cluster was due to MariaDB having to synchronize the database nodes.

Choosing different frameworks might impact the read speeds, but the test shows read speeds were not an issue as they scaled almost perfectly with cluster size. And write speeds are unlikely to improve by changing frameworks.
